app/models/Experiments.php

```diff
@@ -182,6 +179,21 @@ public function updateStatus($status)
     }
 
     /**
+    * Returns if this experiment can be timestamped
+    *
+    * @return bool
+    */
+    public function isTimestampable()
+    {
+        $currentStatus = (int) $this->entityData['category_id'];
+        $sql = "SELECT allow_timestamp FROM status WHERE id = :status LIMIT 1;";
```

Comment:
There is no need for the limit because there is id in the where so you can only ever have one result because id is the primary_key.